tests/governance.py

```diff
@@ -123,6 +124,52 @@ def test_user(network, args, verify=True):
     return network
 
 
+@reqs.description("Validate sample Jinja templates")
+@reqs.supports_methods("log/private")
+def test_jinja_templates(network, args, verify=True):
+    # Note: This test should not be chained in the test suite as it creates
+    # a new user
```

Comment:
Why is this an issue?

Reply:
Looks like it's not! I copied this from the function above, but since we don't use a custom `LoggingTxs` object, and remove the user afterwards, it feels like that shouldn't be a problem for the suite. However, it does rely on a custom arg which doesn't feel worth adding to the suite? So I'll remove the comment, but not chain it.